disnake/message.py

```diff
@@ -955,7 +955,8 @@ def __init__(
 
     def __repr__(self) -> str:
         name = self.__class__.__name__
-        return f"<{name} id={self.id} channel={self.channel!r} type={self.type!r} author={self.author!r} flags={self.flags!r}>"
+        content = self.content[:25] if self.content else None
```

Comment:
```suggestion
        content = (self.content[:22] + "...") if len(self.content) > 25 else self.content
```
This should include an ellipsis when the max length gets exceeded, to indicate that the content is truncated.
Other than that, lgtm - I somehow missed this PR last time.